frontend/lib/letter-of-complaint.tsx

```diff
@@ -11,34 +12,37 @@ import { CenteredPrimaryButtonLink } from './buttons';
 import { SessionProgressStepRoute, RedirectToLatestStep } from './progress-redirection';
 import { Route, Switch } from 'react-router';
 
+export const Welcome = withAppContext((props: AppContextType): JSX.Element => {
+  const { firstName } = props.session;
 
-export function Welcome(): JSX.Element {
   return (
     <Page title="Let's start your letter!">
       <div className="content">
-        <h1 className="title">Let's start your letter!</h1>
+        <h1 className="title">Hi {firstName}, welcome to JustFix.nyc! Let's start your Letter of Complaint.</h1>
```

Comment:
Since this page is currently viewable by folks who aren't logged-in, I think it will read as "Hi, welcome to Justfix.nyc!", which sounds ok.

Uh, that said, we should probably implement #112 at some point.